_includes/nav.html

```diff
@@ -112,16 +97,12 @@
               Community
               <div class="arrow down md theme_denim indicator"></div>
             </div>
-            <div class="accordion-panel"><a target="_blank" rel="noopener noreferrer"
-                href="https://www.kafka-summit.org/">Kafka Summit Conference</a>
-              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/community/">Meetups &amp;
-                Events</a>
-              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/blog/">Blog</a>
-              <a href="https://developer.confluent.io/podcast/">Podcast</a>
-              <a target="_blank" rel="noopener noreferrer"
-                href="https://www.confluent.io/community/ask-the-community/">Ask the Community</a>
-              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/nominate/">Community
-                Catalysts</a>
+            <div class="accordion-panel">
+              <a target="_blank" rel="noopener noreferrer" href="https://www.kafka-summit.org/">Kafka Summit Conference</a>
+              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/community/">Meetups &amp;Events</a>
+              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/community/ask-the-community/">Ask the Community</a>
+              <a target="_blank" rel="noopener noreferrer" href="https://www.confluent.io/nominate/">Community Catalysts</a>
+              <a rel="noopener noreferrer" href="#" style="font-weight: normal">Catalyst spotlight</a>
```

Comment:
Why is there a link to nothing?

Reply:
That's how it's there in the contentful.

Reply:
Removed now.